client/dist/styles.css

```diff
@@ -18,6 +32,12 @@ body {
   font-family: 'Roboto', sans-serif;
 }
 
+/* Compatibility: so safari buttons don't look old and dumb */
```

Comment:
This was moved from the Friend section of the CSS file (near the bottom) to this top common area. You will see it removed starting on line 288 on the left side.